crates/env/src/engine/off_chain/test_api.rs

```diff
@@ -209,6 +209,42 @@ where
     })
 }
 
+/// Transfers value from the caller account to the contract.
```

Comment:
Some doc comments related on how to set the caller and callee accounts in a test would be useful

Reply:
Well, the functions that make this are documented. I mean, if this action (manually setting caller and callee) did not require special docs beforehand (like e.g. [here](https://github.com/paritytech/ink/pull/1379/files/c49d0ffa99091e9350273f0c433fd75e6f0a74bf#diff-d388925fdaa4ef71cf03648e80d5e087cbb248271c13a396dd7d09516e59c93eR202)), why should it require it here. For end-user facing case the example is specially updated in this PR.

Reply:
I'm not sure what you're linking to. 

Either way, we should always be looking to improve our documentation if possible. If it's easy and reasonable to add example usage, we should do it